CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Add OpenTelemetry Context. ([#1617](https://github.com/getsentry/relay/pull/1617))
 - Add `app.in_foreground` and `thread.main` flag to protocol. ([#1578](https://github.com/getsentry/relay/pull/1578))
 - Add support for View Hierarchy attachment_type. ([#1642](https://github.com/getsentry/relay/pull/1642))
+- Add support for `keepalive_timeout` configuration. ([#1645](https://github.com/getsentry/relay/pull/1645))
```

Comment:
Could you, please, move this CHANGELOG entry to  `Features` in `Unreleased` section?

Reply:
done